.github/workflows/govulncheck-action.yml

```
@@ -41,10 +41,11 @@ jobs:
     - name: Lint
       uses: golangci/golangci-lint-action@v8
       with:
+        working-directory: govulncheck-action
         version: v2.1.6
         skip-pkg-cache: true
         skip-build-cache: true
-        args: --config=./govulncheck-action/.golangci.yml --verbose ./govulncheck-action/...
+        args: --config=../govulncheck-action/.golangci.yml --verbose .
```

I didn't know that you can define the `working-directory` also for Linter GHA. In that case, this can be simplified:
```suggestion
        args: --config=.golangci.yml --verbose .
```